app/src/main/res/values-ar/strings.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2" xmlns:tools="http://schemas.android.com/tools">
-    <string name="action_search">بحث</string>
+    <string name="action_search">اسامه</string>
```

Comment:
Putting this in a translator this seems to say "Osama". I guess someone just put their name in the translation field instead of actually translating? :/